src/creature.cpp

```diff
@@ -954,12 +951,12 @@ void Creature::onEndCondition(ConditionType_t)
 
 void Creature::onTickCondition(ConditionType_t type, bool& bRemove)
 {
-	const Tile* tile = getTile();
+	auto tile = getTile();
 	if (!tile) {
 		return;
 	}
 
-	const MagicField* field = tile->getFieldItem();
+	auto field = tile->getFieldItem();
```

Comment:
Could this be `const auto`
```suggestion
	const auto field = tile->getFieldItem();
```